setup.py

```diff
@@ -47,7 +47,7 @@
 ]
 
 # _comp_args = ["-ggdb"]
-_comp_args = ["-O3"] if not ON_WINDOWS else None
+_comp_args = ['-O3', '-g0', '-s'] if not ON_WINDOWS else None
```

Comment:
What is `-s` for here?

Reply:
It behaves similar to stripping afterwards but it's not exactly the same (stripping still reduces size a little bit). -g0 undoes the -g added by setuptools by default.

Reply:
Cool. Might be worth checking if `-Os` instead of `-O3` reduces size any further. Doubt the O3 optimisations are making much of a difference in speed.